jobs/integr8ly/all-tests-executor.yaml

```diff
@@ -150,7 +150,7 @@
         }
 
         def runTests(testPipeline) {
-            buildStatus = build(job: testPipeline, propagate: false, parameters: parameters).result
+            buildStatus = build(job: testPipeline, propagate: false, parameters: jobParams).result
             println "Build finished with ${buildStatus}"
```

Comment:
These both changes are not necessary (just leftover after my experiments) on the other hand this is little bit more clear.

Reply:
I don't mind either way